testing/snapshot-test.md

```diff
@@ -0,0 +1,20 @@
+# Snapshot test
+
+## Introduction
+Snapshot tests are used to test visual regression of presentational components and for reducers in order to avoid deep comparison boilerplate code
+
+## Dos
+- The visual component output has to be relatively small.  
```

Comment:
Snapshots should have a maximum size (e.g 150 LOC) otherwise people gloss over them in code review.